client/my-sites/plans/current-plan/feature-included-card/style.scss

```
@@ -0,0 +1,64 @@
+@import "@wordpress/base-styles/breakpoints";
+
+.feature-included-card__card {
+	width: 240px;
+	display: flex;
+	flex-direction: column;
+	border: 1px solid var(--color-neutral-5);
+	border-radius: 2px;
+	// Assimetric padding same as the designs
```

Nit typo: Assimetric => Asymmetric
```suggestion
	// Asymmetric padding same as the designs
```